tests/unit/string_lib_test.c

```diff
@@ -496,6 +496,76 @@ static void test_string_to_long_compatibility(void)
     // Old function (StringToLongUnsafe) does not allow trailing whitespace
 }
 
+#define ASSERT_INT_EQUAL_STRING_DECIMAL_TO_LONG(str, value_out, expected) \
+    assert_int_equal(StringDecimalToLong(str, &value_out), 0); \
+    assert_int_equal(value_out, expected)
```

Comment:
```suggestion
    assert_int_equal(value_out, expected);
```

Reply:
the omission of `;` was intentional so that compilation fails.
```c
ASSERT_INT_EQUAL(...)
ASSERT_INT_EQUAL(...)
//becomes
//                           V
assert_int(...); assert_int() assert_int(); assert_int()
``` 
And putting a `;` to mimic a func call now means we have a redundant `;`
```c
ASSERT_INT_EQUAL(...);
// becomes
assert_int(...); assert_int(...);;
```
Isnt the macro without the `;` preferable as it forces a more strict code syntax?

Reply:
Okay, I see. I think it's more normal to put all the statements inside a scope `{}`. Let's leave it like this, then.

FYI:

Keep in mind that macros are tricky, and `assert_int_equal()` is also a macro.

What is allowed in terms of semicolons and curly brackets depends a lot on the context, for example, consider creating a function-like macro to put here:

```
for (int i=0; i += my_macro(i); ++i)
...
```